src/Semaphore/SemaphoreTable.css

```diff
@@ -0,0 +1,9 @@
+.SemaphoreTable .table-responsive {
+    border-top: none;
+  }
```

Comment:
These indentation issues keep popping up, any idea why?

Reply:
Probably because on one of my machines, in one of my editors, I've forgotten to configure it properly.  Note to self: Add vim, sublime, and vscode to my Git-stored config files...